core/src/main/java/org/apache/iceberg/actions/RewriteDataFilesCommitManager.java

```diff
@@ -198,9 +199,9 @@ public void offer(RewriteFileGroup group) {
     }
 
     /**
-     * Returns all File groups which have been committed
+     * Returns all file groups results which have been committed
      */
-    public List<RewriteFileGroup> results() {
+    public List<RewriteDataFiles.FileGroupRewriteResult> results() {
```

Comment:
Can you import `FileGroupRewriteResult` rather than using the long name here?

Reply:
Of course, I think the `FileGroupRewriteResult` interface can be taken out separately to solve this problem